### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_ColourContrast: Colour contrast of all the elements on the page is below the required level.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6781]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version:1.10.0-insider (user setup)
Commit: 1ff9a2e
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tools: Accessibility insight for windows
MAS References: MAS1.4.3

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Select the colour theme as "Solarized Light"
7. Expand the server connected in step 2.
8. Right click an on prem database and select Data-tier application wizard from Context menu.
9. Select third radio button.
10. Navigate till next button & hit enter.
11. Launch  Accessibility insight for windows and open the contrast tab.

**Actual:**
On checking the colour contrast ratio of the theme "Solarized Light" for the elements on the page , it is below 4.5:1.

**Similar issue is observed for the following page:**
1.ExtractDacpacSettings
Follow the same steps mentioned above but select the second radio button.

**Expected:**
The colour contrast ration for the elements on the page , should be equal or greater than 4.5:1.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Low-vision user will face difficulty while using application.

**MAS Reference:**
MAS1.4.3-[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

**Attachment for Reference:**
![A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_ColourContrast](https://user-images.githubusercontent.com/24580570/63151281-c3eead00-c026-11e9-962b-162130f4170f.png)

**Comment by udeeshagautam** [comment]
On hold for now. Will chat around high contrast only vs all themes settings.

**Comment by fsteffi** [comment]
Since this bug is not related to the default theme and the ratio of the elements are in proper ratio for default theme closing the bug.
